tiles/src/main/java/com/protomaps/basemap/layers/Landcover.java

```diff
@@ -17,19 +19,42 @@ public class Landcover implements ForwardingProfile.LayerPostProcessor {
   static final Map<String, Integer> sortKeyMapping =
     Map.of("barren", 0, "snow", 1, "crop", 2, "shrub", 3, "grass", 4, "trees", 5);
 
+  public void processNe(SourceFeature sf, FeatureCollector features) {
+    // Daylight landcover uses ESA WorldCover which only goes to a latitude of roughly 80 deg S.
+    // Parts of Antarctica therefore get no landcover = glacier from Daylight.
+    // To fix this, we add glaciated areas from Natural Earth in Antarctica.
+    if (sf.getSourceLayer().equals("ne_10m_glaciated_areas")) {
+      try {
+        Point centroid = (Point) sf.centroid();
+        // Web Mercator Y = 0.7 is roughly 60 deg South, i.e., Antarctica.
+        if (centroid.getY() > 0.7) {
+          features.polygon(LAYER_NAME)
+            .setId(0)
+            .setAttr("kind", "glacier")
+            .setMaxZoom(7)
+            .setMinPixelSize(1.0)
+            .setPixelTolerance(Earth.PIXEL_TOLERANCE);
+        }
+      } catch (GeometryException e) {
+        e.log("Error: " + e);
+      }
+    }
+  }
+
   public void processLandcover(SourceFeature sf, FeatureCollector features) {
     String daylightClass = sf.getString("class");
     String kind = kindMapping.getOrDefault(daylightClass, daylightClass);
 
     // polygons are disjoint and non-overlapping, but order them in archive in consistent way
     Integer sortKey = sortKeyMapping.getOrDefault(daylightClass, 6);
 
-    features.polygon(this.name())
-      .setId(FeatureId.create(sf))
+    features.polygon(LAYER_NAME)
+      .setId(0)
```

Comment:
We may actually want to set a separate ID depending on the landcover class (can do this in a follow up)